loadgen/adapters/vcservice.go

```diff
@@ -78,6 +78,7 @@ func (c *VcAdapter) RunWorkload(ctx context.Context, txStream *workload.StreamWi
 	defer dCancel()
 	g, gCtx := errgroup.WithContext(dCtx)
 	for _, stream := range streams {
+		stream := stream
```

Comment:
The loop variable scope has been fixed since go 1.22 -- https://go.dev/blog/loopvar-preview#the-fix